AmplifyPlugins/Analytics/AWSPinpointAnalyticsPluginIntegrationTests/README.md

```diff
@@ -0,0 +1,26 @@
+## Storage Integration Tests
+
+The following steps demonstrate how to set up Analytics with authenticated access. That's why when you provision analytics at the backend, auth is also provisioned.
+
+
+### Set-up
+
+1. `amplify init`
+
+2. `amplify add analytics`
+
+```perl
+? Select an Analitics provider: Amazon Pinpoint`
+? Provide your pinpoint resource name `Your pinpoint resource name`
+? Apps need autorization to send analytics events. Do you want to allow guests and unauthenticated users to send analytics events? (we recommend you allow this when getting started) `Yes`
+```
+
+3. `amplify push`
+
+[temporary step]: Until Amplify CLI supports adding the auth section into amplifyconfiguation.json, copy `awsconfiguration.json`'s auth section over
+
+4. Copy `amplifyconfiguration.json`  to path "~/Analytics/HostApp/Configuration" and change the name to `AWSPinpointAnalyticsPluginIntegrationTests-amplifyconfiguration.json`
```

Comment:
the path is just inside the Integration test folder, the file reference should be there already, just the actual file is not checked in because of the gitignore. so it should be clear enough to say 
`Copy `amplifyconfiguration.json` over to this folder as `AWSPinpointAnalyticsPluginIntegrationTests-amplifyconfiguration.json`

Reply:
Changed to "Copy `amplifyconfiguration.json` over to `~/AWSPinpointAnalyticPluginIntegrationTests/AWSPinpointAnalyticsPluginIntegrationTests-amplifyconfiguration.json`"

Reply:
i think `~` represents the developer's home directory, like `/Users/<user>/`

Reply:
Ok, what about this "Copy amplifyconfiguration.json over to folder `AWSPinpointAnalyticPluginIntegrationTests` as `AWSPinpointAnalyticsPluginIntegrationTests-amplifyconfiguration.json`"